### PR TITLE
fix:4.4取图失败

### PR DESCRIPTION
**Pull request by wsong910**

### CODE REVIEW

File: android/src/main/java/com/reactnative/ivpusic/imagepicker/activity/PhotoPickerActivity.java [hunk 1]
```diff
@@ -259,16 +259,21 @@ public void onBackPressed() {
     }
 
     private void commit() {
-        if (!photoController.getSelectedPhoto().isEmpty()) {
-
+        if (photoController == null) {
+            return;
+        }
+        final ArrayList<String> selectedPhoto = photoController.getSelectedPhoto();
+        if (selectedPhoto == null) {
```
Comment (greedbell) [inline android/src/main/java/com/reactnative/ivpusic/imagepicker/activity/PhotoPickerActivity.java:266]: Could be changed to 

```
if (selectedPhoto == null || selectedPhoto.isEmpty()) {
    return;
}

```

File: package.json [hunk 2]
```diff
@@ -1,16 +1,36 @@
 {
```
Comment (greedbell) [inline package.json]: revert this file

File: android/src/main/java/com/reactnative/ivpusic/imagepicker/PickerModule.java [hunk 3]
```diff
@@ -249,6 +262,7 @@ public Void call() throws Exception {
 
     @ReactMethod
     public void openCropper(final ReadableMap options, final Promise promise) {
+        //TODO 4.4
```
Comment (greedbell) [inline android/src/main/java/com/reactnative/ivpusic/imagepicker/PickerModule.java:265]: delete useless comment `//TODO 4.4`

File: android/src/main/java/com/reactnative/ivpusic/imagepicker/PickerModuleActivity.java [hunk 4]
```diff
@@ -0,0 +1,58 @@
...
+import com.reactnative.ivpusic.imagepicker.activity.BasePickerActivity;
+
+/**
+ * Created by song on 2018/1/6.
+ */
+
+public class PickerModuleActivity extends BasePickerActivity {
+    static final String REQUEST_CODE_KEY = "requestCode";
+    static final String IS_PICK_KEY = "pick";
+    static final String URI_KEY = "uri";
```
Comment (sohobloo) [inline android/src/main/java/com/reactnative/ivpusic/imagepicker/PickerModuleActivity.java:18]: Why not just use
+import static com.reactnative.ivpusic.imagepicker.PickerModuleActivity.IS_PICK_KEY;
+import static com.reactnative.ivpusic.imagepicker.PickerModuleActivity.REQUEST_CODE_KEY;
+import static com.reactnative.ivpusic.imagepicker.PickerModuleActivity.URI_KEY;

**Pull request merged**